src/devtools/client/inspector/inspector.js

```diff
@@ -903,6 +907,13 @@ class Inspector {
       });
     }
 
+    if (features.newMarkupView) {
+      sidebarPanels.push({
+        id: "markupview",
+        title: INSPECTOR_L10N.getStr("inspector.panelLabel.markupView"),
+      });
+    }
```

Comment:
I wonder if we should just swap markup views when this is on?  Up to you.

Reply:
I am gonna work on that next.